test/controllers/url_controller_test.rb

```diff
@@ -51,41 +45,6 @@ class UrlControllerTest < ActionDispatch::IntegrationTest
     assert_response :unprocessable_content
   end
 
-  test "POST /url - should successfully create a new url when target_url is in request body and is valid" do
```

Comment:
Integration tests (behaviour across different endpoints) has been moved to the `integration` folder.